### ISSUE TITLE
Matplotlib backend does not support multi-edges

### ISSUE THREAD

**Issue by szhorvat** [igraph/python-igraph#395]
**Describe the bug**
The matplotlib backend plots multi-edges on top of each other.

**To reproduce**
```
import matplotlib.pyplot as plt

g = ig.Graph.Formula('0-1-2-3-2',simplify=False)

fig, ax = plt.subplots()
ax.set_aspect('equal', 'box')
ig.plot(
    g,
    target=ax,
    vertex_size=8, edge_alpha=0.3,
)

plt.show()
```

![image](https://user-images.githubusercontent.com/1212871/119015088-e13d7180-b998-11eb-9866-396e3125374f.png)

**Version information**
0.9.1

**Comment by ntamas** [comment]
Note to ourselves: the Cairo backend uses the [`autocurve()`](https://igraph.org/python/doc/api/igraph.html#autocurve) function to assign a "curvature" to each edge in a multiple-edge bundle. We can probably make use of this in the Matplotlib backend as well. Since `autocurve()` is potentially time-consuming, we use it only for graphs with less than 10K edges.

**Comment by iosonofabio** [comment]
I'm pretty sure I messed with autocurve in the mpl backend, at least for loops. Let me have a look

**Comment by ntamas** [comment]
This is now fixed in 0.9.4 so closing.
